### PR TITLE
SNOW-200120 increase ssl coverage

### PR DESCRIPTION
**Pull request by sfc-gh-mkeller**
Deleting some unused code. Consequently removing a now unneeded dependency

### CODE REVIEW

File: src/snowflake/connector/ssl_wrap_socket.py [hunk 1]
```diff
@@ -41,36 +34,6 @@
 """
 FEATURE_OCSP_RESPONSE_CACHE_FILE_NAME = None
 
-
```
Comment (sfc-gh-stakeda) [inline src/snowflake/connector/ssl_wrap_socket.py:44]: Are these for python2? I forgot why I added these...

Comment (sfc-gh-mkeller) [reply src/snowflake/connector/ssl_wrap_socket.py:44]: I'm not sure, let me dig around!

Comment (sfc-gh-mkeller) [reply src/snowflake/connector/ssl_wrap_socket.py:44]: I can't find the reason of why they were added, they were part of the initial release.
But basically this whole commit came from noticing that `ssl_wrap_socket` function was unused and then deleting that exposed a lot of things that were only called from that function.

Comment (sfc-gh-stakeda) [reply src/snowflake/connector/ssl_wrap_socket.py:44]: Looks like this removed the dependency of `ssl_wrap_socket` in this module.
https://github.com/snowflakedb/snowflake-connector-python/commit/6c84a997

**Review by sfc-gh-kwagner — APPROVED** [review]
Nice job!

**Review by sfc-gh-stakeda — APPROVED** [review]

**Pull request merged**